### PR TITLE
les: fix CHT field in nodeInfo

### PR DESCRIPTION
**Pull request by zsfelfoldi**
This PR fixes the empty CHT field of admin.nodeInfo in light server mode

### CODE REVIEW

File: les/commons.go [hunk 1]
```diff
@@ -76,18 +76,30 @@ func (c *lesCommons) makeProtocols(versions []uint) []p2p.Protocol {
...
+		sections /= light.CHTFrequencyClient / light.CHTFrequencyServer
+	}
+
 	if sections2 < sections {
 		sections = sections2
-		sectionHead = sectionHead2
 	}
 	if sections > 0 {
 		sectionIndex := sections - 1
+		sectionHead := c.bloomTrieIndexer.SectionHead(sectionIndex)
```
Comment (karalabe) [inline les/commons.go]: if sections2 < sections, this will refer to non existent section head.

Comment (karalabe) [reply les/commons.go]: or the other way around, either way, the head will be invalid if section != section2.

Comment (zsfelfoldi) [reply les/commons.go]: No. sectionIndex is the last section which both indexers have so both of them should have the section head for it.

Comment (karalabe) [reply les/commons.go]: Doh

**Pull request merged**